src/sinol_make/commands/run/__init__.py

```diff
@@ -368,70 +381,176 @@ def print_view(output_file=None):
 		return program_groups_scores
 
 
-	def validate_subtasks(self):
-		print("Validating subtasks...")
-		if 'subtasks' not in self.config.keys():
-			print(util.bold(util.color_red('Subtasks description not defined in config.yml.')))
-			exit(1)
+	def calculate_points(self, results):
+		points = 0
+		for group, result in results.items():
+			if group != 0 and group not in self.config["scores"]:
+				print(util.error(f'Group {group} doesn\'t have points specified in config file.'))
+				exit(1)
+			if result == "OK" and group != 0:
+				points += self.config["scores"][group]
+		return points
+
 
-		programs = []
-		for subtask in self.config["subtasks"]:
+	def validate_config(self, programs):
+		print("Validating config...")
```

Comment:
Is this function needed when we also validate all of its config in the `validate_expected_scores`?

Reply:
I think we should remove this function, it doesn't add much

Reply:
Removed function in 82c01ce5595419b14df61f5cd7ee7938837b4c00